ui/public/config.json

```diff
@@ -59,6 +59,32 @@
     "jp": "label.japanese.keyboard",
     "sc": "label.simplified.chinese.keyboard"
   },
+  "userDoc": [
+    {
+      "title": "Documentation",
+      "text": "CloudStack documentation website",
+      "link": "https://docs.cloudstack.apache.org/en/latest/",
```

Comment:
This would lead to the latest documentation page, irrespective of the backend ACS version. Can we use `$store.getters.features.cloudstackversion` to get the version and redirect to the correct version of the docs.

Reply:
this is admin configurable, so I've kept it as latest - an admin might define env specific website or completely change the list of links (for ex. have an internal/link for support tickets etc)

Reply:
Ok got it..